docs/release-notes.md

```diff
@@ -2,6 +2,23 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
 
+## Nov 13, 2023
+
+*Proton:*
+  * [JDBC driver](https://github.com/timeplus-io/proton-java-driver) for Proton is now available. Check out our [example](https://github.com/timeplus-io/proton/tree/develop/examples/jdbc) for how to connect to Proton with DBeaver. We also submitted a PR to [Pulse UI](https://github.com/timestored/pulseui/pull/139).
+  * An experimental [ODBC driver](https://github.com/timeplus-io/proton-odbc) for accessing Proton as a data source is also available.
+  * An experimental `shuffle by` clause has been added. Currently, it only works for historical queries, and it will support streaming queries soon. The key use case for `shuffle by` is to support high cardinality `group by`. To learn more about this advanced feature, join the discussion in our [Slack community](https://timeplus.com/slack).
+  * By default, the backfill from historical store for time travel/rewind is enabled. For example `select * from stream where _tp_time>now()-14d` will load data from historical store for last 14 days, even for data not available in streaming storage. If you prefer the previous behavior, you can add `setting enable_backfill_from_historical_store=false` to the streaming SQL.
```

Comment:
this default setting is changed since 1.3.19. Need to mention the version number, otherwise ppl think this is always enabled (for backfill from hist store)

Reply:
also `settings` not `setting`